account_statement_completion/models/completion_rules.py

```diff
@@ -28,6 +29,29 @@ class StatementCompletionRule(models.Model):
     _name = "account.statement.completion.rule"
     _description = 'Account Statement Completion Rule'
 
+    DEFAULT_VAL = """
+                    # Available variables:
+                    #-------------------------------
+                    # stmts_vals: Values of the statements as a list of dict
+                    # stmt_line: Values of the statement line as a dict
+                    # line_amount: stmt_line amount
+                    # ref: stmt_line reference
+                    # env: environment
+                    # AccMove: Odoo model "account.move"
+                    # AccMoveLine: Odoo model "account.move.line"
+                    # Partner: Odoo model "res.partner".
+                    # Payment: Odoo model "account.payment"
+                    # PaymentLine: Odoo model "account.payment.line"
```

Comment:
I would remove all these variables from the evaluation context. They can be inferred when needed in the code using the `env` variable.

Reply:
Does it change a lot performance wise ? If not giving an easy accessible base for the "users" seems quite convenient to me.

Reply:
The users are still technical skilled people that should know about the "env" in Odoo. Using that is easier than having to know every existing variable in the context. Please make the change.